### DISCUSSION TITLE
How to show object name in the anyOf droplist

### DISCUSSION THREAD

**Discussion by xli** [stoplightio/elements#1858] [Q&A] [answered]
Hi, 

Great project, thanks for making it open source.

I tried to render my openapi specification by elements
`https://elements-demo.stoplight.io/?spec=https://raw.githubusercontent.com/diem/diem/main/api/doc/openapi.yaml#/operations/get_transactions`

But found when there is `anyOf` is used, the droplist is always showing same text (`object` or `array[objects]`), see the following screenshort:
![image](https://user-images.githubusercontent.com/17474/141662963-8e39841f-2722-4b3a-a313-3f1c377770bd.png)

I found couple resolved issues related to showing `anyOf` element with title, but could not figure out a way to show meaningful text in the droplist when `anyOf` is used.

**Comment by xli** [comment, marked as answer]
Probably some mistakes I made when trying out elements, today, when I try again with `title` fields setup, it works.

**Comment by mnaumanali94** [comment]
It was indeed an issue. We have fixed it recently.
